static/src/js/components/Banner/__tests__/Banner.test.js

```diff
@@ -15,43 +14,47 @@ function setup(overrideProps = {}) {
     ...overrideProps
   }
 
-  const enzymeWrapper = shallow(<Bannner {...props} />)
+  render(<Bannner {...props} />)
 
   return {
-    enzymeWrapper,
     props
   }
 }
 
 describe('Bannner component', () => {
   test('should render self', () => {
-    const { enzymeWrapper } = setup()
+    setup()
+
+    expect(screen.getByRole('heading', { name: 'Banner Title' })).toBeInTheDocument()
+    expect(screen.getByRole('heading', { name: 'Banner Title' })).toHaveTextContent('title')
+
+    expect(screen.getByRole('paragraph', { name: 'Banner Message' })).toBeInTheDocument()
+    expect(screen.getByRole('paragraph', { name: 'Banner Message' })).toHaveTextContent('message')
 
-    expect(enzymeWrapper.find('h2.banner__title').text()).toEqual('title')
-    expect(enzymeWrapper.find('p.banner__message').text()).toEqual('message')
+    expect(screen.getByRole('img', { name: 'High Alert Icon' })).toBeInTheDocument()
   })
 
-  test('clicking the close button calls onClose', () => {
-    const { enzymeWrapper, props } = setup()
+  test('clicking the close button calls onClose', async () => {
+    const user = userEvent.setup()
+    const { props } = setup()
 
-    const button = enzymeWrapper.find(Button)
-    button.simulate('click')
+    const closeBtn = screen.getByRole('button', { name: 'close' })
+    await user.click(closeBtn)
 
     expect(props.onClose).toBeCalledTimes(1)
```

Comment:
```suggestion
    expect(props.onClose).toBeCalledTimes(1)
    expect(props.onClose).toBeCalledWith()
```
I think we should still check that its called with no arguments

Reply:
It's not called with no arguments, that was enzyme. Because this is RTL, it has an event passed into the onClick.

Reply:
We should check that it was called with an event that we expect in that case